### PR TITLE
yarn-support

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - set CI=true
+  - npm install -g yarnpkg
+  - yarn
+
 test_script:
   - node --version
-  - npm --version
```
Comment (vinnymac) [inline appveyor.yml]: I would leave this line, as npm is still being used in places. I would also add `- yarn --version` below it.

File: .travis.yml [hunk 2]
```diff
@@ -23,8 +23,8 @@ addons:
 
 install:
   - export CXX="g++-4.8"
-  - npm install -g npm@latest
-  - npm install
+  - npm install -g yarnpkg
```
Comment (vinnymac) [inline .travis.yml]: According to the alt install methods listed on the docs site https://yarnpkg.com/en/docs/install

It looks like `npm i -g yarn` is the one we want here.

File: appveyor.yml [hunk 3]
```diff
@@ -1,25 +1,32 @@
...
-install:
-  - ps: Install-Product node $env:nodejs_version
-  - set CI=true
-  - npm install -g npm@latest
-  - set PATH=%APPDATA%\npm;%PATH%
-  - npm install
+
+cache:
+  - node_modules
+
```
Comment (vinnymac) [inline appveyor.yml]: Add `$HOME/.yarn-cache` for yarn specific caching

**Comment by amilajack** [comment]
I think I made this after creating https://github.com/yarnpkg/yarn/issues/920 😂

**Comment by amilajack** [comment]
Proper documentation was added:

https://yarnpkg.com/en/docs/install_ci

**Pull request merged**